src/listen.js

```diff
@@ -135,11 +135,10 @@ module.exports = function(defaultFuncs, api, ctx) {
                 }
                 return;
               }
-
               break;
             case 'buddylist_overlay':
               // TODO: what happens when you're logged in as a page?
-              if(!ctx.globalOptions.updatePresence) {
+              if(!ctx.globalOptions.batchPresence) {
```

Comment:
Added minor adjustments and fixed as you correctly pointed out updatePresence. Told ya I was prone to typos 😄